tests/unittests/classification/test_accuracy.py

```diff
@@ -60,6 +66,53 @@ def _sklearn_accuracy_binary(preds, target, ignore_index, multidim_average):
         return np.stack(res)
 
 
+class TestAccuracy(MetricTester):
```

Comment:
let's not instantiate full `MetricTester` class just for testing two specific cases

Reply:
implemented in https://github.com/Lightning-AI/metrics/pull/1448/commits/199a71afb30aceee4e82c43dd0b6a7a7f9486cec